docs/refs.bib

```diff
@@ -425,4 +425,326 @@ @article{yeh2009comparisons
+  year = {2022},
+  month = jan,
+  publisher = {{BMJ}},
+  volume = {29},
+  number = {1},
+  pages = {e100459},
+  author = {Laura Sikstrom and Marta M Maslej and Katrina Hui and Zoe Findlay and Daniel Z Buchman and Sean L Hill},
+  title = {Conceptualising fairness: three pillars for medical algorithms and health
+		                        equity},
+  journal = {{BMJ} Health Care Inform}
+}
+
+@article{krieger2012population,
+  doi = {10.1111/j.1468-0009.2012.00678.x},
+  url = {https://doi.org/10.1111/j.1468-0009.2012.00678.x},
+  year = {2012},
+  month = dec,
+  publisher = {Wiley},
+  volume = {90},
+  number = {4},
+  pages = {634--681},
+  author = {Nancy Krieger},
+  title = {Who and What Is a {\textquotedblleft}Population{\textquotedblright}? Historical Debates,  Current Controversies,  and Implications for Understanding {\textquotedblleft}Population Health{\textquotedblright} and Rectifying Health Inequities},
+  journal = {Milbank Quarterly}
+}
+
+@article{wallace2011interactions,
+  doi = {10.1093/carcin/bgr066},
+  url = {https://doi.org/10.1093/carcin/bgr066},
+  year = {2011},
+  month = apr,
+  publisher = {Oxford University Press ({OUP})},
+  volume = {32},
+  number = {8},
+  pages = {1107--1121},
+  author = {T. A. Wallace and D. N. Martin and S. Ambs},
+  title = {Interactions among genes,  tumor biology and the environment in cancer health disparities: examining the evidence on a national and global scale},
+  journal = {Carcinogenesis}
+}
+
+@article{eknoyan2007adolphe,
+  doi = {10.1093/ndt/gfm517},
+  url = {https://doi.org/10.1093/ndt/gfm517},
+  year = {2007},
+  month = aug,
+  publisher = {Oxford University Press ({OUP})},
+  volume = {23},
+  number = {1},
+  pages = {47--51},
+  author = {G. Eknoyan},
+  title = {Adolphe Quetelet (1796 1874) the average man and indices of obesity},
+  journal = {Nephrology Dialysis Transplantation}
+}
+
+@misc{bbc2016dutch,
+	author = {BBC News},
+	title = {{D}utch men revealed as world's tallest --- bbc.com},
+	howpublished = {\url{https://www.bbc.com/news/science-environment-36888541}},
+	year = {2016},
+	note = {[Accessed 19-Jul-2023]},
+}
+
+@misc{endocrine2009widely,
+  author = {The Endocrine Society},
+  title = {Widely Used Body Fat Measurements Overestimate Fatness In African-Americans, Study Finds},
+  howpublished={\url{www.sciencedaily.com/releases/2009/06/090611142407.htm}},
+  year={2009},
+  journal={ScienceDaily},
+  note = {[Accessed 19-Jul-2023]},
+}
+
+@article{racette2003obesity,
+  doi = {10.1093/ptj/83.3.276},
+  url = {https://doi.org/10.1093/ptj/83.3.276},
+  year = {2003},
+  month = mar,
+  publisher = {Oxford University Press ({OUP})},
+  volume = {83},
+  number = {3},
+  pages = {276--288},
+  author = {Susan B Racette and Susan S Deusinger and Robert H Deusinger},
+  title = {Obesity: Overview of Prevalence,  Etiology,  and Treatment},
+  journal = {Physical Therapy}
+}
+
+@misc{psychologytoday2016AdolpheQuetelet,
```

Comment:
The format should be firstauthorYEARtitle1word

Reply:
Not looking forward to updating all refs, but fair enough :D